tests/php/SiteTreeSubsitesTest.php

```diff
@@ -1,33 +1,55 @@
 <?php
 
+namespace SilverStripe\Subsites\Tests;
+
+use Page;
+use SilverStripe\CMS\Controllers\CMSMain;
+use SilverStripe\CMS\Controllers\ModelAsController;
+use SilverStripe\CMS\Model\ErrorPage;
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Control\Director;
+use SilverStripe\Control\Session;
+use SilverStripe\Core\Config\Config;
+use SilverStripe\Core\Convert;
+use SilverStripe\Dev\TestOnly;
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Security\Member;
+use SilverStripe\SiteConfig\SiteConfig;
+use SilverStripe\Subsites\Extensions\SiteTreeSubsites;
+use SilverStripe\Subsites\Model\Subsite;
+use SilverStripe\Subsites\Pages\SubsitesVirtualPage;
+use SilverStripe\Versioned\Versioned;
+use SilverStripe\View\SSViewer;
+
 class SiteTreeSubsitesTest extends BaseSubsiteTest
 {
-    protected static $fixture_file = 'subsites/tests/SubsiteTest.yml';
 
-    protected $extraDataObjects = array(
+    public static $fixture_file = 'subsites/tests/php/SubsiteTest.yml';
+
+    protected $extraDataObjects = [
```

Comment:
This is now `protected static $extra_dataobjects = []` and each class in it should be fully qualified